tests/crypto/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+
+project(NONE)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+FILE(GLOB app_sources src/*.c)
+
+if(REDUCED_TEST_SUITE)
+# Quick reduced case: Run only a selection of test vectors,
+# found in the single file test_vectors.c
+FILE(GLOB test_vectors test_cases/test_vectors.c)
+else()
+# Normal case: Run everything in all test vector files.
+FILE(GLOB test_vectors test_cases/test_vectors_*.c)
+endif()
+
+if(DISABLE_SLOW_VECTORS)
+# Some test vectors are much slower than others.
+# These will not be compiled if this definition is added.
+# Do a search of CRYPTO_LONG_RUNNING_VECTORS_DISABLE amongst the
+# test_vectors_* files to see which vectors are affected.
+add_definitions(-DCRYPTO_LONG_RUNNING_VECTORS_DISABLE)
+endif()
```

Comment:
This should be configured through Kconfig.

Reply:
Moved to Kconfig in 7aee7a9